src/components/computerVsHumanGameComponent/computerVsHumanGameComponent.js

```diff
@@ -51,9 +43,34 @@ class ComputerVsHumanGameComponent extends Component {
       let current_player = this.computer.symbol;
       let next_player = this.state.human;
       let marks = this.state.board.marks();
+
+      this.updateBoardState(marks, next_player, current_player);
+    }
+
+    updateBoardState = async (marks, next_player, current_player) => {
       this.setState({
-        gameStatus: await PhoenixApi.getStatus(marks, next_player, current_player),
+        gameStatus: await PhoenixApi.getStatus(marks, next_player, current_player)
       });
+
+      if(this.state.gameStatus === Statuses.IN_PROGRESS && next_player === this.computer.symbol){
+        this.setState({
+          gameStatus: Statuses.COMPUTER_THINKING
+        });
+        setTimeout( () => {this.computerMove();}, 1000);
+      }
+
+      if (this.state.gameStatus !== Statuses.IN_PROGRESS || this.state.gameStatus !== Statuses.COMPUTER_THINKING) {
+        this.highlightWinners(marks);
+      }
+    }
+
+    highlightWinners = async (marks) => {
+      let data = await PhoenixApi.getWinningIndices(marks);
+      let winningCells = data.index_list;
+      winningCells.forEach((index) => {
+        let indexString = parseInt(index);
+      document.getElementById(indexString).classList.add('highlight');
+      })
```

Comment:
again, we are both fetching and highlighting. Let's spit this into smaller methods.